.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+labels: Enhancement
+
+---
+
+**Describe the problem/challenge you have**
+[A description of the current limitation/problem/challenge that you are experiencing.]
+
+
+**Describe the solution you'd like**
+[A clear and concise description of what you want to happen.]
+
+
+**Anything else you would like to add:**
+[Miscellaneous information that will assist in solving the issue.]
+
+
+**Environment:**
```

Comment:
Do we need setup details in feature request?

Reply:
yes. it will be good to have.